cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/edges/Extensions.kt

```diff
@@ -80,3 +80,10 @@ fun <
     val edge = edgeProperty.call(this)
     return edge.delegate()
 }
+
+/** Returns the first edge with the given [name] or `null` if no such edge exists. */
+operator fun <NodeType : Node, EdgeType : Edge<NodeType>> EdgeList<NodeType, EdgeType>.get(
```

Comment:
Is this new function used in this PR at all? :confused:

Reply:
Yes it it. https://github.com/Fraunhofer-AISEC/cpg/pull/1965/files#diff-71de8469911ec3c4f5162303939f8730ee3e8513676d0b125218e58beb113b16R1724

Reply:
Oops. Ok.